docs/source/features/packaging_output_models.md

```diff
@@ -22,57 +21,96 @@ Zipfile packaging will generate a ZIP file which includes 3 folders: `CandidateM
 
 Each `BestCandidateModel` folder will include model file/folder. The folder also includes a json file which includes the Olive Pass run history configurations since input model, a json file with performance metrics and a json file for inference settings for the candidate model if the candidate model is an ONNX model.
 
-##### Inference config file
-The inference config file is a json file including `execution_provider` and `session_options`. e.g.:
+#### SampleCode
+Olive will only provide sample codes for ONNX model. Sample code supports 3 different programming languages: `C++`, `C#` and `Python`. And a code snippet introducing how to use Olive output artifacts to inference candidate model with recommended inference configurations.
 
+#### Models rank JSON file
+A file that contains a JSON list for ranked model info across all accelerators, e.g.:
 ```
-{
-    "execution_provider": [
-        [
-            "CPUExecutionProvider",
-            {}
-        ]
-    ],
-    "session_options": {
-        "execution_mode": 1,
-        "graph_optimization_level": 99,
-        "extra_session_config": null,
-        "inter_op_num_threads": 1,
-        "intra_op_num_threads": 64
-    }
-}
+[
+    {
+        "rank": 1,
+        "model_config": {
+            "type": "ONNXModel",
+            "config": {
+                "model_path": "path/model.onnx",
```

Comment:
I suppose it should be relative path in package.zip. right?

Reply:
Oh I will immediately start working on that once this PR is merged.